src/messagingService/index.js

```diff
@@ -0,0 +1,45 @@
+import messaging from '@react-native-firebase/messaging'
+import firebase from 'firebase'
+import 'firebase/functions'
+
+export const checkForPermissions = async () => {
+  const enabled = await messaging().hasPermission()
+  if (!enabled) {
+    try {
+      await messaging().requestPermission()
+    } catch (error) {
```

Comment:
I initially thrown errors but because of how are called from login, I removed those lines.
It could make the user login fail after user signed in, so I think it should be console log until we rework login screen

Reply:
If the user doesn't give permissions, the app should keep going on. It will only affect notifications